docs/setting-your-IDE.md

```diff
@@ -0,0 +1,91 @@
+## The basics
+
+If you are interested in being able to quickly debug and develop Mailchimp Open Commerce, consider installing these useful tools to help you.
+
+- You should use [nvm](https://github.com/nvm-sh/nvm) to install and manage NodeJS.
+- Visual Studio Code is a preferred text editor but you can use Sublime Text, Emacs, Vim or any other editor of your choice.
+
+## Browser Extensions
+
+### Chrome
+
+- [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi)
```

Comment:
I think this list should add some context as to why you would want to install these and we might want to break these down a little bit into types of extensions.